web/profiles/custom/yalesites_profile/config/sync/responsive_image.styles.full_width.yml

```diff
@@ -18,7 +18,7 @@ dependencies:
     - image.style.max_width_800
     - image.style.max_width_960
 id: full_width
-label: 'Full Width'
+label: 'Image - Max Width'
```

Comment:
I think it's probably a good idea to rename this to match the term used on the frontend. Can you also rename the machine name though? (Since we're still in early alpha, it's okay to introduce breaking changes) I'm not sure where else this is used at the moment though, so that might require some digging